pages/understanding-json-schema/reference/type.md

```diff
@@ -175,3 +175,64 @@ types. For example, numeric types have a way of specifying a numeric
 these validation keywords are described along with each of their
 corresponding types in the following chapters.
+
+## Format[#format]
+
+The `format` keyword conveys semantic information for values that may be difficult or impossible to describe using JSON Schema. Typically, this semantic information is described by other documents. The JSON Schema Validation specification defines several formats, but this keyword also allows schema authors to define their own formats.
+
+For example, because JSON doesn't have a "DateTime" type, dates need to be encoded as strings. `format` allows the schema author to indicate that the string value should be interpreted as a date. By default, `format` is just an annotation and does not affect validation.
+
+Optionally, validator [implementations](../../learn/glossary#implementation) can provide a configuration option to enable `format` to function as an assertion rather than just an annotation. That means that validation fails when, for example, a value with a `date` format isn't in a form that can be parsed as a date. This allows values to be constrained beyond what other tools in JSON Schema, including [Regular Expressions](../../understanding-json-schema/reference/regular_expressions), can do.
+
+> Implementations may provide validation for only a subset of the built-in formats or do partial validation for a given format. For example, some implementations may consider a string an email if it contains an `@`, while others might perform additional checks for other aspects of a well-formed email address.
+
+The JSON Schema specification has a bias toward networking-related formats, likely due to its roots in web technologies. However, custom formats may also be used if the parties exchanging the JSON documents share information about the custom format types. A JSON Schema validator will ignore any format type it does not understand.
+
+### Built-in Formats
+
+It should be noted that `format` is not limited because it only defines a specific set of valid values for formats. Users may define their own custom keywords to work with any specific data type, such as `integer`, `double`, `float`, etc. Below, we cover the commonly used `string` formats specified in the JSON Schema specification.
+
+#### Dates and Times
+
+Dates and times are represented in [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). RFC 3339 is a specification from the Internet Engineering Task Force (IETF).
+
+- `"date-time"`: Date and time together, for example, `2018-11-13T20:20:39+00:00`.
+- `"time"`: <StarInline label="New in draft 7" /> Time, for example, `20:20:39+00:00`.
+- `"date"`: <StarInline label="New in draft 7" /> Date, for example, `2018-11-13`.
+- `"duration"`: <StarInline label="New in draft 2019-09" /> A duration as defined by the [ISO 8601 ABNF for "duration"](https://datatracker.ietf.org/doc/html/rfc3339#appendix-A). For example, `P3D` expresses a duration of 3 days.
```

Comment:
We link to RFC3339 here, but say that it's from ISO8601.  I think the text is good, but maybe we should link to 8601 directly rather than to the 3339 appendix. 

The text in the spec is:

> The duration format is from the ISO 8601 ABNF as given in Appendix A of RFC 3339.

@mwadams thoughts?

Reply:
Doesn't ISO keep their standards behind a paywall? Linking to ISO8601 may not be an option.

Reply:
Oh, yeah.  That's dumb.  I guess that's why we copied everything relevant into that appendix.